app/controllers/api/v4/invoices_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Api
+  module V4
+    class InvoicesController < ApplicationController
+      def index
+        @event = authorize(Event.find_by_public_id(params[:event_id]) || Event.friendly.find(params[:event_id]), :invoice_list?)
+        @invoices = @event.invoices.order(created_at: :desc)
```

Comment:
```suggestion
        @event = Event.find_by_public_id(params[:event_id])
        @invoices = authorize(@event.invoices.order(created_at: :desc))
```

https://github.com/hackclub/hcb/blob/367cf2d85d213e21295c8aade7407a1d8027f13a/app/controllers/invoices_controller.rb#L10-L11
https://github.com/hackclub/hcb/blob/edb0fe4e9e7aaeee2c4a33fee2e366d21e109806/app/policies/invoice_policy.rb#L4-L14